views/publication/summary.layout.gohtml

```diff
@@ -1,134 +1,137 @@
 {{$mainFile := .Publication.MainFile}}
-<div class="list-group-item-inner">
-    <div class="list-group-item-main u-min-w-0">
-        <div class="c-thumbnail-and-text align-items-start d-block d-lg-flex">
+<div class="w-100">
+    <div class="c-thumbnail-and-text align-items-start w-100">
+        {{if eq .UserRole "curator"}}
```

Comment:
Non curators do not require a thumbnail? Or is it because not everyone is capable of seeing a thumbnail anyway?

Reply:
This is indeed the intended behaviour. I'll be happy to discuss features/product decisions outside of a PR.